### DISCUSSION TITLE
cdc是不是只能在主库上执行

### DISCUSSION THREAD

**Discussion by stgztsw** [apache/flink-cdc#1547] [Q&A]
请问，官方文档里面说需要开启REPLICATION SLAVE, REPLICATION CLIENT等权限，这些权限是不是只能在主库上开启，所以cdc只能运行在主库上

**Comment by lykpass** [comment]
![image](https://user-images.githubusercontent.com/48595535/189062657-a1a1d920-07cc-4f77-9453-c6fc0de175a1.png)

**Reply by haodonp** [reply]
postgresql 库呢？

**Reply by stgztsw** [reply]
收到，谢谢
